service/kafka-plugin/pom.xml

```diff
@@ -10,12 +11,12 @@
     <properties>
         <maven.compiler.source>17</maven.compiler.source>
         <maven.compiler.target>17</maven.compiler.target>
-        <spring-kafka.version>3.0.10</spring-kafka.version>
+        <spring-kafka.version>3.1.4</spring-kafka.version>
         <kafka-clients.version>3.5.0</kafka-clients.version>
-        <spring-boot-version>3.1.11</spring-boot-version>
-        <commons-collections4.vresion>4.4</commons-collections4.vresion>
-        <jupiter.version>5.3.1</jupiter.version>
-        <camel.version>4.2.0</camel.version>
+        <spring-boot.version>3.2.5</spring-boot.version>
+        <commons-collections4.version>4.4</commons-collections4.version>
+        <jupiter.version>5.10.2</jupiter.version>
+        <camel.version>4.5.0</camel.version>
```

Comment:
Are these upversions needed? If yes, have we run scans against kafka brokers to make sure it still works?

Reply:
Yeah, they are necessary. I'll see about testing it.